pkg/autoscaling/metrics/store/store.go

```diff
@@ -0,0 +1,70 @@
+// Copyright 2020 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package metrics
+
+import "time"
+
+// Store provides interfaces to query metrics
+type Store interface {
```

Comment:
Is it Store a good name?

Reply:
How about Querier? I think Store is a little bit confusing with the Store that holds regions.

Reply:
better than Store